src/test/migrations/steps/20240919123658-step-update-cycle-props-published-date-status.ts

```diff
@@ -59,18 +59,12 @@ export default async () => {
   }
 
   await client.query(`
-      do $$
-      begin
-        if exists (
-          select 1
-          from information_schema.columns
-          where table_name='assessment_cycle' and column_name='published'
-        ) then
+        do $$
+        begin
```

Comment:
Simplify the migration by removing double check for `published` column
We already check for `published` above, so no need for double check here.

It was required previously when running this under `public`-migration